src/components/views/rooms/EventTile.tsx

```diff
@@ -1641,6 +1642,12 @@ export default class EventTile extends React.Component<IProps, IState> {
             }
         }
     }
+
+    render() {
```

Comment:
To make the future PR of mine easier to write:
```suggestion
    public render() {
```